transform/voxel_around_line_test.go

```diff
@@ -161,6 +199,30 @@ func TestFitClearanceAroundExtendedSpatialID04(t *testing.T) {
 
 }
 
+func TestFitClearanceAroundExtendedSpatialID05(t *testing.T) {
```

Comment:
This test will pass without merging this pull request.
Modify this test and prove that the issue is resolved.

Reply:
TestFitClearanceAroundExtendedSpatialID01 should fail before pull request https://github.com/trajectoryjp/spatial_id_go/blob/58638b928b671281bd4f4a261d0ff4eb6ab31a83/transform/voxel_around_line_test.go#L106

Reply:
@buscandoaverroes 
I checked it out.
Thank you!